packages/avatar/README.md

```diff
@@ -137,6 +143,16 @@ import { Avatar } from '@spectrum-web-components/avatar';
 </sp-tab-panel>
 </sp-tabs>
 
-## Accessibility
+### States
+
+#### Generic avatars
+
+Use branded generic avatars when a user has not set their avatar image. These images are designed to be abstracted from all genders, locales, and cultures.
```

Comment:
This feels less like a state, but I'm not sure it's necessarily an option either. Thoughts?